### DISCUSSION TITLE
Minimum system requirements for running Monika?

### DISCUSSION THREAD

**Discussion by kabua** [hyperjumptech/monika#1032] [Q&A] [answered]
What are the minimum system requirements for running Monika on Lunis using Docker?

Would the following be enough?

- 1 vCore CPU
- 512 MB RAM
- 10 GB SSD

**Comment by sapiderman** [comment, marked as answer]
Hi @kabua,

If you can get docker up and running, then yes Monika in docker mode should run all right. Let us know if you find any issues!

FYR: We've tried it on Digital Ocean's basic droplet, 512MB/1vCPU with no problems. However if you have a large number of probes, you might hit a limit with the small RAM.

- Best Regards, Monika Team

**Reply by kabua** [reply]
Okay, thanks for the insight.
